sdk/webpubsub/azure-messaging-webpubsub/src/samples/java/com/azure/messaging/webpubsub/WebPubSubJavaDocCodeSnippets.java

```diff
@@ -232,7 +232,7 @@ public void groupSendToAllVarArgs() {
         WebPubSubGroup adminGroup = null;
 
         // BEGIN: com.azure.messaging.webpubsub.webpubsubgroup.sendToAll#String
-        adminGroup.sendToAll("Hello world!");
+        adminGroup.sendToAll("{\"message\": \"Hello world!\"}");
```

Comment:
just a thought, for convenice we could add JSON string input Helper class which helps to create the JsonString maybe.

Reply:
Jackson, Json-P and other libraries already have several helper methods to build the JSON. So, we don't need to roll our own. This also what we do in LLC.